demos/demo-minimal-js/index.js

```diff
@@ -92,7 +98,11 @@ export function logOut() {
 
 export function incomingCall() {
   window.setTimeout(() => {
-    cti.incomingCall();
+    cti.incomingCall({
+      createEngagement: "true",
```

Comment:
Considering our goal of HubSpot owned engagements, should we be providing the ability to opt out of engagement creation?

Reply:
I agree that we should enforce HubSpot owned engagements but this would gate against creating a duplicate engagement if a provider sends us an `incomingCall` event but also creates the engagement on their side.

Reply:
Makes sense!